cypress/locators/configurable-items/index.js

```diff
@@ -1,3 +1,4 @@
 export const draggableItemByText = text => cy.get('.common-input__label').contains(text).parent().parent()
   .find('.icon-drag_vertical');
-export const draggableItemByPosition = position => cy.get('.carbon-configurable-items__items-wrapper').find(`li:nth-child(${position})`).find('label');
+export const draggableItemByPosition = position => cy.get('.carbon-configurable-items__items-wrapper')
```

Comment:
maybe we should make the locators in locators.js file ?
the same question is for draggable-context

Reply:
agreed, done